### DISCUSSION TITLE
Command Line access to change Wi-Fi Passphrase

### DISCUSSION THREAD

**Discussion by CyberLake87** [RaspAP/raspap-webgui#1276] [Q&A]
I have managed to change the wifi password for RaspAP and now cannot access the interface. This is not admin/secret pair, this is what was originally ChangeMe.
Can this be altered at the Command Line, I thought I had notes but I cannot find them anywhere,
TIA

**Comment by billz** [comment]
Edit hostapd.conf with `sudo nano /etc/hostapd/hostapd.conf` and set the `wpa_passphrase` value.
Give the service a kick for the changes to take effect: `sudo systemctl restart hostapd.service`.
